### ISSUE TITLE
some how the app stopped downloading

### ISSUE THREAD

**Issue by alaa777** [patricker/Pulse#3]

**Comment by alaa777** [comment]
what do you need to diagnose the issue it just stopped downloading wallpapers

**Comment by patricker** [comment]
So no wallpapers are downloading at all huh? Let me check my end to see if it's the same.

**Comment by patricker** [comment]
Interesting. Yes, Wallhaven is failing to download, though "Preview" works fine. Interesting.

**Comment by patricker** [comment]
Yes, it looks like Wallhaven has tweaked the website. I can load the URL's in my browser, but am getting an HTTP 403 response code right now.

**Comment by alaa777** [comment]
yeah sorry i have been off the internet for a few days
so is there a fix?

**Comment by alaa777** [comment]
did the website some how disable the ability to apps to download?

**Comment by alaa777** [comment]
any updates?

**Comment by patricker** [comment]
I was debugging this yesterday. What's really interesting is the "Preview" button works. It doesn't seem to have any trouble connecting to Wallhaven, running the search, and getting the list of results back. Even downloading the thumbnails works fine.  It's just the full sized picture that fails...

**Comment by alaa777** [comment]
hmmmm
hope there is a fix or a workaround

**Comment by alaa777** [comment]
so no fix on the horizon ?

**Comment by alaa777** [comment]
should i stop waiting? :)

**Comment by patricker** [comment]
I wish I could say, "I'll have it fixed soon!" But truth be told... I just don't have time to work on this anymore :(
You've been a long time user, and at this point are probably the last user.

**Comment by alaa777** [comment]
oh well thank you so much and wish you good :)
